src/Point/Point.cs

```diff
@@ -95,26 +97,50 @@ public async Task HandleWebSocket(WebSocket webSocket)
             await _connection.SendTimeoutMessage();
         }
 
+        /// <summary>
+        /// Sets the timeout flag for the next command.
+        /// </summary>
+        public void EnableTimeout(){

```

Comment:
Can we establish a uniform naming convention for methods that have a direct effect vs. methods that change the behavior of the next point command?
Also, does a timeout have to have an effect on the (degraded) point position, as per the spec? Or is it possible for the point to still reach an end position after the timeout has been reported?

Reply:
If we could annotate the appropriate portions of the code with requirements identifiers, that would be a plus.